tests/fixers/test_http_headers.py

```diff
@@ -0,0 +1,105 @@
+from __future__ import annotations
+
+from django_upgrade.data import Settings
+from tests.fixers.tools import check_noop
+from tests.fixers.tools import check_transformed
+
+settings = Settings(target_version=(4, 2))
+
+
+def test_transform_constructor_call():
+    check_transformed(
+        """\
+        from django.test import RequestFactory
+        RequestFactory("/", HTTP_HOST="test.com")
+        """,
+        """\
+        from django.test import RequestFactory
+        RequestFactory("/", headers={"host": "test.com"})
+        """,
+        settings,
+    )
+
+
+def test_transform_constructor_call_multiple():
+    check_transformed(
+        """\
+        from django.test import Client
+        c = Client("/", HTTP_HOST="test.com", headers={"user-agent": "example"}, HTTP_ACCEPT_LANGUAGE="fr-fr")
+        """,  # noqa: E501
+        """\
+        from django.test import Client
+        c = Client("/", headers={"user-agent": "example", "host": "test.com", "accept-language": "fr-fr"},)
```

Comment:
Trailing comma here, is that an issue? 🤔 

Or we remove it when the `headers` is the last argument.